src/main/java/org/zalando/planb/provider/CassandraConfig.java

```diff
@@ -19,6 +20,7 @@ Session initializeCassandra() {
         final Cluster.Builder builder = Cluster.builder();
 
         builder.addContactPoints(cassandra.getContactPoints().split(","));
+        builder.withAddressTranslator(new EC2MultiRegionAddressTranslator());
```

Comment:
@harti2006 sounds good, but I'm really not up to speed with Spring.  Would make sense to sit together to make this change?